### PR TITLE
allow using whitespaces in object names

### PR DESCRIPTION
**Pull request by schurzi**

### CODE REVIEW

**Comment by schurzi** [comment]
fixes #52

File: plugins/module_utils/icinga.py [hunk 1]
```diff
@@ -9,6 +9,8 @@
 
 from ansible.module_utils.urls import fetch_url
 from ansible.module_utils.six import iteritems
+from ansible.module_utils._text import to_text
```
Comment (rndmh3ro) [inline plugins/module_utils/icinga.py:12]: Looking at the module here: https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/_text.py
It says `.. warn:: Use ansible.module_utils.common.text.converters instead.`

File: tests/integration/targets/icinga/roles/icinga/tasks/special_whitespace.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
```
Comment (rndmh3ro) [inline tests/integration/targets/icinga/roles/icinga/tasks/special_whitespace.yml:1]: I don't want to manually manage tests (manually handling this file is enough: `tests/integration/targets/icinga/roles/icinga/tasks/main.yml`).

Can you please add this example (or change the existing one so it contains whitespace) into the examples here: `plugins/modules/icinga_service.py`? We could use a "minimal" example with only the required fields and whitespace.

**Review by rndmh3ro — CHANGES_REQUESTED** [review]

File: plugins/module_utils/icinga.py (hunk 1 above)
Comment (schurzi) [reply plugins/module_utils/icinga.py:12]: that's funny, I took this piece of code from ansible-galaxy XD

Comment (schurzi) [reply plugins/module_utils/icinga.py:12]: https://github.com/ansible/ansible/blob/3e9943bc5e7a9cd393757aa8100d7fed80bd316e/lib/ansible/galaxy/api.py

**Review by rndmh3ro — APPROVED** [review]

**Pull request merged**